assets/app/data/crowbar/services/crowbar-backup.factory.js

```diff
@@ -1,19 +1,24 @@
 (function() {
 
     angular
-        .module('crowbarData.upgrade')
-        .factory('upgradeBackupFactory', upgradeBackupFactory);
+        .module('suseData.crowbar')
+        .factory('crowbarBackupFactory', crowbarBackupFactory);
 
-    upgradeBackupFactory.$inject = ['$q', '$http', '$filter', 'COMMON_API_V2_HEADERS'];
+    crowbarBackupFactory.$inject = ['$q', '$http', '$filter', 'COMMON_API_V2_HEADERS'];
     /* @ngInject */
-    function upgradeBackupFactory($q, $http, $filter, COMMON_API_V2_HEADERS) {
+    function crowbarBackupFactory($q, $http, $filter, COMMON_API_V2_HEADERS) {
         var factory = {
             create: createBackup,
```

Comment:
Yeah, a bit redundant but as the var+function below changed their names to include backup, I was thinking more about consistency between both methods `crowbarBackupFactory().createBackup()` and `crowbarBackupFactory().getBackup()`